sandbox/Sandbox.jsx

```diff
@@ -27,6 +27,12 @@ export default class Sandbox extends React.Component {
         super(props);
 
         const { config: configOverride, data } = sandboxData;
+
+        defaultConfig.d3.onCollide = node => {
```

Comment:
Ideally, this function would receive colliding nodes, still I know that d3 does not provide that, not even the `node` that you declared there is passed, in fact, the `onCollide` seems to be receiving some transform delta value which IMHO is not of big use here :confused:

Reply:
This is supposed to be a tunable function that exists on your graph config. It will allow you to programatically determine what the collision force radius is for any one of your graph nodes. For example, if the node is a switch, then you could set the collision force radius to be smaller and the node is a non-switch then you could set the collision force radius to be larger. This would give your overall graph a better / more fine tuned layout on initial and subsequent loads. You initially asked me to console.log in this method similar to that of onClickNode. I agree, the current example does not do a good job of demonstratrating how to use the onCollide method. Do you have a better suggestion?

Reply:
Since this is a function that exists on the config, which is defined by the user of this framework, we may be better off excluding this from the sandbox but including it in the [documentation](https://goodguydaniel.com/react-d3-graph/docs/index.html)